docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  mongodb:
+    image: mongo:3
+
+  redis:
+    image: redis:latest
+
+  http:
+    image: nginx:latest
+
+  dynamodb:
+    image: deangiberson/aws-dynamodb-local:latest
+
+  await:
+    image: saltside/await
+    build: .
+    environment:
+      DYNAMO_DB_URL: http://dynamodb:8000
+      AWS_ACCESS_KEY_ID : "test"
```

Comment:
adam these variables need to be present in environment. I cannot find an option in aws-cli where i can pass it directly. Only thing i find is via using profiles. Do you know the options which i can use?